magiccube/cube.py

```diff
@@ -315,6 +315,42 @@ def reverse_history(self, to_str=False):
 
         return reverse
 
+    @property
+    def get_facelet_colors_in_kociemba_order(self) -> str:
```

Comment:
```suggestion
    def get_kociemba_facelet_colors(self) -> str:
        """Return the string representation of the cube facelet colors in Kociemba format (ex: WWWWWWWWWRRRRRRRRRGGGGGGGGGYYYYYYYYYOOOOOOOOOBBBBBBBBB)."""
```
Leaving here the suggestion to rename the method to use the same pattern as other methods and have a docstring to help other API users.

Reply:
Suggestions applied, I wasn't inspired for the name of theses method, so I prefered a very long but explicit name for me.